spec/commands/v2/put_content_spec.rb

```diff
@@ -226,7 +226,7 @@
           described_class.call(payload)
           Commands::V2::Publish.call(content_id: content_id, update_type: "minor")
 
-          regex = /Content item user_facing_version=(\d+) and locale=en for content item=#{Regexp.quote(content_id)} conflicts/
+          regex = /Content item state=draft and locale=en for content item=#{Regexp.quote(content_id)} conflicts/
```

Comment:
Kinda weird this needs changing? not sure whats causing it - maybe it's concealing a bug?